### PR TITLE
dev-util/webstorm: version bump to 2017.2.4

### PR DESCRIPTION
**Pull request by strahlc**
Package-Manager: Portage-2.3.6, Repoman-2.3.1

Fixes Mantis #614840

- version bump to 2017.2.3
- added desktop entry
- cleanup ebuild

This PR supersedes PR #5548

### CODE REVIEW

**Comment by strahlc** [comment]
@mgorny 
Sorry for the new PR. I've done a rebase mistake which make the old PR unreadable and confusing. But I've incorporated all your comments from PR #5548.
I now bump to webstorm 2017.2.3

File: dev-util/webstorm/webstorm-2017.2.3.172.3968.27.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+DESCRIPTION="JavaScript IDE for client- and server-side development with Node.js"
+HOMEPAGE="http://www.jetbrains.com/webstorm"
+SRC_URI="http://download.jetbrains.com/${PN}/WebStorm-$(get_version_component_range 1-3).tar.gz"
+
+LICENSE="WebStorm WebStorm_Academic WebStorm_Classroom WebStorm_OpenSource WebStorm_personal"
+SLOT="$(get_major_version)"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=virtual/jdk-1.7"
```
Comment (monsieurp) [inline dev-util/webstorm/webstorm-2017.2.3.172.3968.27.ebuild]: Is Java really needed here?

Comment (bderagon) [reply dev-util/webstorm/webstorm-2017.2.3.172.3968.27.ebuild]: yes, the entire ide for webstorm is written in java

**Comment by strahlc** [comment]
it seems that @bderagon has merged this commit weeks ago without closing the PR or mentioning the author.

**Pull request closed without merging**